### ISSUE TITLE
Microchip (Tier 1) assembler recipes ONLY uses integrated logic circuits, instead of any low circuit

### ISSUE THREAD

**Issue by Marzule-90** [FTBTeam/FTB-Interactions#1160]
## Modpack Version: 
2.0.7

## Bug Report:
Recipe only uses integrated logic circuits in the assembler, instead of any low circuit.

## Expected Behavior/Suggestions:
Suggestion: allow recipe to accept any low circuit for automation late game.

**Comment by Possseidon** [comment]
Yeah, currently the three different basic circuits are used to differentiate the OC microchips tiers.

It would probably be better to differentiate this with a numbered integrated circuit and allow all basic circuits in all the recipes.

**Comment by Possseidon** [comment]
Actually, basically a duplicate of #1122
